src/overlays/actors/ovl_En_Nimotsu/z_en_nimotsu.c

```diff
@@ -28,25 +31,96 @@ const ActorInit En_Nimotsu_InitVars = {
     (ActorFunc)EnNimotsu_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80BE1FB0 = {
-    { COLTYPE_NONE, AT_NONE, AC_NONE, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_2, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK4, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_NONE, OCELEM_ON, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_NONE,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_2,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK4,
+        { 0x00000000, 0x00, 0x00 },
+        { 0x00000000, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_NONE,
+        OCELEM_ON,
+    },
     { 10, 30, 0, { 0, 0, 0 } },
 };
 
-#endif
+void EnNimotsu_UpdateCollision(EnNimotsu* this, PlayState* play) {
+    Collider_UpdateCylinder(&this->actor, &this->collider);
+    CollisionCheck_SetOC(play, &play->colChkCtx, &this->collider.base);
+    Actor_UpdateBgCheckInfo(play, &this->actor, 32.0f, 30.0f, 0.0f, 4);
+}
+
+void EnNimotsu_Init(Actor* thisx, PlayState* play) {
+    EnNimotsu* this = THIS;
+
+    Collider_InitCylinder(play, &this->collider);
+    Collider_InitAndSetCylinder(play, &this->collider, &this->actor, &sCylinderInit);
+
+    this->dustTimer = 10;
+    this->actor.gravity = -0.5f;
+
+    Actor_SetScale(&this->actor, 0.01f);
+}
+
+void EnNimotsu_Destroy(Actor* thisx, PlayState* play) {
+    EnNimotsu* this = THIS;
+
+    Collider_DestroyCylinder(play, &this->collider);
+}
+
+void EnNimotsu_Update(Actor* thisx, PlayState* play) {
+    s32 pad;
+    EnNimotsu* this = THIS;
+
+    Vec3f dustPosition;
```

Comment:
Please remove the empty line
```suggestion
    EnNimotsu* this = THIS;
    Vec3f dustPosition;
```